tests/test_general_validators/conftest.py

```diff
@@ -17,3 +17,10 @@ def origin_repo():
     origin_repo_dir = 'test_fixtures/general_repo_origin'
     git.Repo.init(origin_repo_dir)
     return LocalRepositoryInfo(origin_repo_dir)
+
+
+@pytest.fixture(scope='session')
+def temp_dir_for_tests(tmpdir_factory):
```

Comment:
I and @Melevir have agreed that this approach to fixtures has some problems:
1. It limits the reusability of the fixtures
2. It degrades the readability of them
3. It creates the potential for hard-to-detect file name clashes (since the folder is session-wide).

So, while this is an interesting approach, we discourage it for the time being. Could you please save the fixtures as normal files?

Reply:
Ok! I'll fix it

Reply:
For future me: this method doesn't even work, since the new file isn't added to the repository and is simply ignored by the test.